pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -302,15 +302,31 @@ def select_channel(self, channel_name):
         """
         self.write(f"INST:SEL '{channel_name}'")
 
-    # active_channel = Instrument.control("INST?", "INST %s", "To get the active channel")
+    @property
+    def active_channel(self):
+        return self.values("INST?")[0]
+
+    @active_channel.setter
+    def activate_channel(self, channel):
+        availabel_channels = [chan for chan in self.available_channels.keys()]
+        channel = strict_discrete_set(channel, availabel_channels)
+        self.write(f"INST '{channel}'")
+
+    def view_mode(self, view="SPL"):
+        """To get a MultiView of the open channels or return to the last active channel
+        Args:
```

Comment:
pymeasure uses the standard dosctring style of sphinx, https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

I recommend installing the vscode extension "Autodocstring" and configure it accordingly. It gives you a template for the docstrings.

Reply:
Resolved via [9d73b1d](https://github.com/bleykauf/pymeasure/pull/4/commits/9d73b1d02c1df0a713dcb7e6d7fef6921f07e2e9)